dawn/src/dawn/IIR/Extents.h

```diff
@@ -40,6 +40,7 @@ class Extent {
   /// @name Constructors and Assignment
   /// @{
   Extent(int minus, int plus) : minus_(minus), plus_(plus) {}
+  Extent(int extent) : Extent(extent, extent) {}
```

Comment:
Is the correct terminology for the parameter here `extent` or `offset`? I would say offset, as it reflects this https://github.com/MeteoSwiss-APN/dawn/blob/25bc24b899dc8eb4436a455af772b0924a6df3c0/dawn/src/dawn/IIR/Extents.h#L219